src/components/map/OpportunityMap.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export const Opportunity: React.FunctionComponent = () => {
+  return (
+    <div>
+        MapSelector
```

Comment:
Should this be `OpportunityMap`, and therefore, did you forget to create `MapSelector.tsx`?

Reply:
We no longer require the map selector we think. Because the rover selector hold the state of the rover, and display map depends on the rover state. We think we dont need the rover seletor.